x/identity/client/rest/rest.go

```diff
@@ -0,0 +1,104 @@
+package rest
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/tendermint/spn/x/identity/types"
+)
+
+const (
+	MethodGet  = "GET"
+	MethodPost = "POST"
+)
+
+var (
+	UsernameRoute            = fmt.Sprintf("custom/%s/%s", types.QuerierRoute, types.QueryUsername)
```

Comment:
@ltacker Is there a reason to use `custom`? Just curious.

Reply:
I actually needed to remove them for the REST route but you must put this prefix for the querier route when it's a custom module.